ld/api_expand.go

```diff
@@ -359,7 +359,11 @@ func (api *JsonLdApi) expandObject(activeCtx *Context, activeProperty string, ex
 		var expandedValue interface{}
 		// 7.3)
 		if expandedProperty == "" || (!strings.Contains(expandedProperty, ":") && !IsKeyword(expandedProperty)) {
-			continue
+			if activeCtx.options != nil && activeCtx.options.SafeMode {
+				return NewJsonLdError(InvalidInput, "property didn't expand into absolute IRI or keyword")
```

Comment:
I've looked at [jsonld.js implementation](https://github.com/digitalbazaar/jsonld.js/blob/ed4918abba9bb3f63bde336d066de9f4cc5732f7/lib/expand.js#L473). It uses a non-standard error code `"invalid property"`.  I would propose to adopt the same approach and add a new JsonLdError called `InvalidProperty` into _non-spec related errors_ [here](https://github.com/piprate/json-gold/blob/18603dd0c38589d1bb06166a730c47bb776395d9/ld/errors.go#L87) and use it instead of InvalidInput.